metadata/network.go

```diff
@@ -17,22 +17,30 @@
 
 package metadata
 
+import "github.com/ethereum/go-ethereum/common"
+
 // NetworkDefinition structure holds all parameters which describe particular network
 type NetworkDefinition struct {
-	DiscoveryAPIAddress string
-	BrokerAddress       string
+	DiscoveryAPIAddress     string
+	BrokerAddress           string
+	EtherClientRPC          string
+	PaymentsContractAddress common.Address
 }
 
 // TestnetDefinition defines parameters for test network (currently default network)
 var TestnetDefinition = NetworkDefinition{
 	"https://testnet-api.mysterium.network/v1",
 	"testnet-broker.mysterium.network",
+	"https://ropsten.infura.io",
```

Comment:
without user key? Will be request limited?

Reply:
Currently there is no limitation of request to infura, with or without key. In this case if such problem arrises - user can always specify infura link with api key embedded.